modules/specifications/lib/specs.ts

```diff
@@ -0,0 +1,19 @@
+export interface SpecIt {
+  type: 'it';
+  name: string;
+}
+
+export interface SpecDescribe {
+  type: 'describe';
+  name: string;
+  children: (SpecDescribe | SpecIt)[];
+}
+
+export interface SpecFile {
+  type: 'file';
+  name: string;
+  children: (SpecDescribe | SpecIt)[];
+}
+
+// prettier-ignore
+export const specifications: SpecFile[] = [/* SPEC_FILES_REPLACE_BY_WEBPACK */];
```

Comment:
Hopefully this is clear enough that Webpack specifically replaces this string with the real specs.